### PR TITLE
{devel,lib}[GCC/13.2.0] poppler v24.04.0, texlive v20230313

### PR DESCRIPTION
**Pull request by Louwrensth**
(almost created using eb --new-pr)

### CODE REVIEW

File: easybuild/easyconfigs/t/texlive/texlive-20230313-GCC-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+]
+checksums = ['d97bdb3b1903428e56373e70861b24db448243d74d950cdff96f4e888f008605']
+
+dependencies = [
+    ('X11', '20231019'),
+    ('libpng', '1.6.40'),
+    ('libGLU', '9.0.3'),
+    ('Perl', '5.38.0'),
+    ('HarfBuzz', '8.2.2'),
+    # TODO ('poppler', '23.09.0'),
```
Comment (bedroge) [inline easybuild/easyconfigs/t/texlive/texlive-20230313-GCC-13.2.0.eb:35]: I suppose this has to use the included easyconfig?
```suggestion
    ('poppler', '24.04.0'),
```

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/7d82dd206f8c973571957167ff2e358b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Louwrensth!

**Pull request merged**